rust/worker/src/system/types.rs

```diff
@@ -36,7 +36,7 @@ pub(crate) trait Component: Send + Sized + Debug + 'static {
     fn runtime() -> ComponentRuntime {
         ComponentRuntime::Inherit
     }
-    async fn on_start(&mut self, ctx: &ComponentContext<Self>) -> () {}
+    async fn on_start(&mut self, _ctx: &ComponentContext<Self>) -> () {}
```

Comment:
this is not right, this is a trait definition, ctx is not used by any consumer, that doesn't mean the trait definition should be empty.